sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -757,6 +757,51 @@ if (contextVersion > 1) {
         function(target) {
             return gl.getIndexedParameter(target, 0);
     });
+
+    debug("");
+    debug("Test getQueryParameter");
+    var buffer = gl.createBuffer();
+    gl.bindBuffer(gl.TRANSFORM_FEEDBACK_BUFFER, buffer);
+    gl.bufferData(gl.TRANSFORM_FEEDBACK_BUFFER, 64, gl.DYNAMIC_DRAW);
+    var query = gl.createQuery();
+    gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, buffer);
+    var requestAnimationFrame = window.requestAnimationFrame ||
+	                        window.mozRequestAnimationFrame ||
+			        window.webkitRequestAnimationFrame ||
+			        window.msRequestAnimationFrame;
+    var cancelRAF = window.cancelAnimationFrame ||
+	            window.mozCancelAnimationFrame ||
+		    window.webkitCancelAnimationFrame ||
+		    window.msCancelAnimationFrame;
+    var requestId;
+    function stopAnimation() {
+        cancelRAF(requestId);
+    }
+    function animate() {
+        gl.beginQuery(gl.TRANSFORM_FEEDBACK_PRIMITIVES_WRITTEN, query);
+        gl.beginTransformFeedback(gl.TRIANGLES);
+        gl.drawArrays(gl.POINTS, 0, 3);
+        gl.endTransformFeedback();
+        gl.endQuery(gl.TRANSFORM_FEEDBACK_PRIMITIVES_WRITTEN);
+        if(gl.getQueryParameter(query, gl.QUERY_RESULT_AVAILABLE) == true)
+           stopAnimation();
+        else
+           requestId = requestAnimationFrame(animate);
+    }
+    animate();
+    shouldBe('gl.getQueryParameter(query, gl.QUERY_RESULT_AVAILABLE)', 'true');
```

Comment:
This code is still incorrect. The shouldBe call will execute immediately after the call to animate(), even if a requestAnimationFrame callback is registered.